src/drift-monitor.ts

```diff
@@ -59,7 +59,7 @@ export class DriftMonitor extends Construct {
         (props.stackNames === undefined || (props.stackNames!.length === 0))) {
       throw new Error('Must provide at least one stack');
     }
-    if (props.runEvery !== undefined && props.runEvery!.toMinutes() < 1) {
+    if (props.runEvery !== undefined && props.runEvery!.toSeconds() < 60) {
```

Comment:
```suggestion
    if (props.runEvery !== undefined && props.runEvery.toSeconds() < 60) {
```

Reply:
TS/TSLint is smart! There are a couple of more places I need to fix then, updated PR.